docs/modules/ROOT/pages/Introduction.adoc

```diff
@@ -72,9 +72,11 @@ To verify your pwa is compliant to the provided Dockerfile, you can use the Dock
 You just need to change the template file in such way, that your custom env parameters are set accordingly to you `.env` file.
 
 Once you have updated your template just rename the file to `Dockerfile`.
-If you then execute `docker build .` and the image could be built successfully, 
+If you then execute `docker build -t my-pwa .` and the image could be built successfully, 
 your application will be compliant with the e-Spirit build pipeline.
```

Comment:
The application will be compliant with the e-Spirit build pipeline, if the image can be build **and** can be started locally. 
The way it is worded now, there may be some misunderstandings.